src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -347,18 +345,12 @@ winrt::hstring Profile::ExpandedBackgroundImagePath() const
 
 winrt::hstring Profile::EvaluatedStartingDirectory() const
 {
-    const auto path{ StartingDirectory() };
-    const auto pathString{ path.as<winrt::Windows::Foundation::IReference<winrt::hstring>>() };
-    if (pathString == nullptr)
+    if (_StartingDirectory)

```

Comment:
don't we need to check the inheritance tree, in case the user set it in `p.defaults`?